src/main/java/core/model/TranslationFacility.java

```diff
@@ -0,0 +1,37 @@
+package core.model;
+
+import static java.util.Objects.requireNonNullElseGet;
+
+public class TranslationFacility {
+
+    private static final String INITIAL_TRANSLATOR = Translator.LANGUAGE_NO_TRANSLATION;
+
+    private static Translator translator;
+    static {
+        translator = Translator.load(INITIAL_TRANSLATOR);
+    }
+
+    private TranslationFacility() {
+        throw new UnsupportedOperationException("This is a utility class and cannot be instantiated");
```

Comment:
> The Exception is probably overkill, and does not fit with the existing pattern.

It makes clear that an instance of the class is not supported there is no intention to create an instance because it is only provided as a namespace.

Think of a class that is a singleton (regardless the questions if singletons are a good idea) when the constructor is private normally.

The thrown exception marks the class clearly as a utility class that is no singleton.

Reply:
OK, then if it's the new pattern for singletons in HO, this needs to be applied to all.  We are open to new styles and new approaches, but consistency in my mind is important.  Also, just to note this will naturally go away with Kotlin `object` declaration.

Reply:
> OK, then if it's the new pattern for singletons in HO, this needs to be applied to all. We are open to new styles and new approaches, but consistency in my mind is important. Also, just to note this will naturally go away with Kotlin `object` declaration.

It is not a pattern for singletons but utility classes (namespaces in C++ e.g.).
A utility class is a class that provides static functions that are grouped together under the name of the class.
The class itself has a private constructor and unlike to singletons the private constructor is never invoked.
The exception is only to make that clear.

Singletons normally have a private constructor that is invoked in the `instance` function.

Reply:
Sorry, that's what I was meaning to say, “non-instantiable utility classes” rather than “singleton.”  The consistency argument still stands, regardless of the incorrect terminology.

Reply:
No, problem.

Please note that the project is not consistent in several places. ;-)

I will create another small refactor PR if you want.

Reply:
Yes, for sure. :-)
Thanks, Sebastian, appreciate the effort!!